app/controllers/api/v1/rents_controller.rb

```diff
@@ -0,0 +1,28 @@
+module Api
+  module V1
+    class RentsController < ApiController
+      include Wor::Paginate
+      def index
+        render_paginated Rent
+      end
+
+      def create
+        rent = Rent.new(rent_params)
+
+        if rent.save
+          render json: { message: 'The rent was created successfully',
```

Comment:
I recomend you to render more simple responses, the only data you will need is rent, so you can do `render json: rent` directly, the status code will expose if the resource was created successfully or not